pkg/utils/external_resource_reader.go

```diff
@@ -21,11 +21,11 @@ import (
 )
 
 // ReadExternalResource reads data streams from external resources. Currently implemented:
-// - '@' for STDIN
+// - '-' or '@' for STDIN
```

Comment:
Usually I have been using `-` as the token to indicate that data should be read from stdin. But for this particular case, in https://github.com/Kuadrant/kuadrantctl/pull/43 I intentionally I switched from `-` to `@` and I cannot remember why :cry:. I guess we will find out soon if there was a good reason.

Reply:
yeah, was unsure why `@` (have seen that used occasionally too, but a bit rarer). Left there in case there was a good reason